### ISSUE TITLE
Option to hide system shortcuts of menu panel

### ISSUE THREAD

**Issue by narzb** [The-Panacea-Projects/Gnomenu#35]
Option to hide this part :
![screenshot_608_3](https://cloud.githubusercontent.com/assets/8238816/11986335/a4823bc0-a9ce-11e5-921d-a6adc7730ff5.png)

**Comment by The-Panacea-Projects** [comment]
Yes this certainaly can be looked at for a future release, are you able to submit a patch to do this?

**Comment by narzb** [comment]
It's not my main skill. But I will try.

**Comment by The-Panacea-Projects** [comment]
Okay appreciated, keep us posted on how it goes.
